### PR TITLE
Initial import of code

### PR DESCRIPTION
**Pull request by petter-fidesmo**
- Create basic set up for the project
- Keep original abstractions with the Transceiver and Card interfaces
- Set up publishing to marmeladburk

### CODE REVIEW

**Comment by petter-fidesmo** [comment]
@slomo This is an initial import of the Android NFC code. Let's see if we can reformat this into something that useful for not only us, but anyone working with NFC on Android.

File: project/build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sbt.version=0.13.7
```
Comment (slomo) [inline project/build.properties]: sbt 0.13.8 is available since march

File: android/src/main/java/nordpol/TagDispatcher.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    @TargetApi(Build.VERSION_CODES.KITKAT)
+    public void disableExclusiveNfc() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+            disableReaderMode();
+        } else {
+            disableForegroundDispatch();
+        }
+    }
+
+    public void interceptIntent(Intent intent) {
```
Comment (slomo) [inline android/src/main/java/nordpol/TagDispatcher.java]: I am wondering whether we should add some mean to signal if the intent was intercepted (meaning already processed)

File: android/build.sbt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+android.Plugin.androidBuildAar
+
+platformTarget in Android := "android-22"
+
+name := "nordpol-android"
+
+// Do not append Scala versions to the generated artifacts
+crossPaths := false
+
+// This forbids including Scala related libraries into the dependency
```
Comment (slomo) [inline android/build.sbt]: I think this comment should rather be: "Prevents the scala stdlib from beeing included automatically."

**Comment by slomo** [comment]
Left a few minor comments. Other that that feel free to merge.

**Comment by petter-fidesmo** [comment]
@slomo like that?

**Comment by slomo** [comment]
:+1:

**Pull request merged**